search/quality_eval.go

```diff
@@ -539,3 +569,83 @@ func WriteToCsv(path string, records [][]string) error {
 	}
 	return nil
 }
+
+func getLatestUidByFilters(filters []Filter, db *sql.DB) (string, error) {
```

Comment:
Nice!

Please test this function, it is fragile, i.e., expected to change with time.

Super! Now after you get UID it is as if it was a standard expectation, at some section, right?

Reply:
Tests added